docs/_docs/windows.md

```diff
@@ -81,6 +90,7 @@ the site generation process. It can be done with the following command:
 ```sh
 $ chcp 65001
 ```
+#### Time-Zone Management (Chocolatey method)
```

Comment:
... Did we not just discuss that it doesn't apply to RubyInstaller and Bash on Windows? I haven't tried chocolatey so I'm not going to delete a section of a tutorial if it might break the method..? Sorry?